### DISCUSSION TITLE
Configuration des unités équivalentes

### DISCUSSION THREAD

**Discussion by mquandalle** [publicodes/publicodes#400] [Ideas]
La proposition #199 concerne à la fois la définition des unités au pluriel (traité dans #307) et les unités équivalentes dont le but est :

- [ ] gérer les synonymes : `1 an = 1 année`
- [ ] gérer les conversions automatiques : `1 pouce = 2,54 cm`
- [ ] gérer les groupes de mille : `1 k€ = 1000 €`, `1 ko = 1000 octets`, `1 MJ = 1 000 000 joules`

**Comment by Clemog** [comment]
Une première tentative dans https://github.com/betagouv/publicodes/pull/374 :)
